packages/wrangler/src/config/validation.ts

```diff
@@ -128,10 +128,17 @@ export function normalizeAndValidateConfig(
 		);
 	}
 
+	const isDispatchNamespace = typeof args["dispatch-namespace"] === "string";
```

Comment:
should we check for an empty string here? so smth like 
```
const isDispatchNamespace = typeof args["dispatch-namespace"] === "string" && args["dispatch-namespace"].trim() !== ""

Reply:
Yeah good shout